documentation/src/docs/security/securing-distributed-mode-cluster.md

```diff
@@ -26,7 +26,7 @@ limitations under the License.
 
 ## Introduction
 
-In the [distributed mode](../deployment/deployment.md#pravega-modes) of running a Pravega cluster, each service runs
+In the [distributed mode](../deployment/deployment.md#Pravega Modes) of running a Pravega cluster, each service runs
```

Comment:
This link is broken, it does not render as a link.

Reply:
I have checked and made the changes as (../deployment/deployment.md#pravega-modes) 
is not pointing any ware checked in deployment.md file there is no such tag (#pravega-modes)
so replaced with #pravega-modes and this is working checked locally.